rosbag2_tests/test/rosbag2_tests/record_fixture.hpp

```diff
@@ -69,14 +65,29 @@ class RecordFixture : public TemporaryDirectoryFixture
     rclcpp::shutdown();
   }
 
+  std::string get_bag_file_name(int split_index) const
+  {
+    std::stringstream bag_file_name;
+    bag_file_name << "bag_" << split_index;
+
+    return bag_file_name.str();
+  }
+
+  rcpputils::fs::path get_bag_file_path(int split_index)
```

Comment:
```suggestion
  rcpputils::fs::path get_bag_file_path(int split_index) const
```

Reply:
Unfortunately [rcpputils::fs::path::operator/](https://github.com/ros2/rcpputils/blob/a098746ce33f78f29f64be46b5d559ceaf407fa5/include/rcpputils/filesystem_helper.hpp#L164) is not const

Reply:
I see - note that this implementation is wrong, operator/ should be a function and not a class method so in C++17 this problem would not appear. Anyway, this sgtm for this commit.